### PR TITLE
counter for responses

### PR DESCRIPTION
**Pull request by james-bowers**
creates `current_count` and `clear_current_count` routes for seeing how many responses of particular status codes are returned. 

Useful for RESFRAME-2745

### CODE REVIEW

**Review by woodyblah — APPROVED** [review]

File: lib/origin_simulator.ex [hunk 1]
```diff
@@ -19,6 +20,20 @@ defmodule OriginSimulator do
     |> send_resp(200, Poison.encode!(msg))
   end
 
+  get "/clear_current_count" do
+    Counter.clear()
+
+    conn
+    |> put_resp_content_type("text/html")
+    |> send_resp(200, "<p>Counter cleared</p>")
```
Comment (samfrench) [inline lib/origin_simulator.ex]: I wonder if this should be more of a JSON API.

**Review by samfrench — APPROVED** [review]

**Review by JoeARO — APPROVED** [review]

File: lib/origin_simulator.ex [hunk 2]
```diff
@@ -19,6 +20,20 @@ defmodule OriginSimulator do
     |> send_resp(200, Poison.encode!(msg))
   end
 
+  get "/clear_current_count" do
```
Comment (ettomatic) [inline lib/origin_simulator.ex:23]: probably a `post` would be more appropriate but for sure less convenient

Comment (james-bowers) [reply lib/origin_simulator.ex:23]: that was my thinking too... but i went with convenience

File: recipes/demo.json [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{
+    "origin": "https://www.bbc.co.uk",
```
Comment (ettomatic) [inline recipes/demo.json]: do we need the additional recipe in the repo?

File: lib/mix/upload_recipe.ex [hunk 4]
```diff
@@ -0,0 +1,19 @@
+defmodule Mix.Tasks.UploadRecipe do
+  use Mix.Task
```
Comment (ettomatic) [inline lib/mix/upload_recipe.ex:2]: ❤️

File: lib/origin_simulator.ex (hunk 2 above)
Comment (ettomatic) [reply lib/origin_simulator.ex:23]: convenience 🆚 conventions :)

**Comment by ettomatic** [comment]
I think it only needs to move the recipe from `/recipes` to `/examples` and it's ready to merge! :)

**Pull request merged**